src/SFA.DAS.Roatp.CourseManagement.Web/Views/AddAStandard/ViewAddedStandardDetails.cshtml

```diff
@@ -0,0 +1,169 @@
+﻿@using SFA.DAS.Roatp.CourseManagement.Web.Models.Standards
```

Comment:
What is this for? There is already a ReviewAndSaveStandard view in place

Reply:
I put that in and then realised the page already existed, and overlooked removing it. Removed now.